spec/support/active_record.rb

```diff
@@ -1,6 +1,6 @@
 require 'database_cleaner'
 
-ActiveRecord::Base.establish_connection(adapter: 'sqlite3', database: 'file::memory:?cache=shared', pool: 10)
+ActiveRecord::Base.establish_connection(adapter: 'sqlite3', database: 'file::memory:?cache=shared', pool: 1)
```

Comment:
It me be interesting to explain here that test suite doesn't run properly with more than one database connection, otherwise it does not find tables